src/main/frontend/schema/handler/common_config.cljc

```diff
@@ -0,0 +1,83 @@
+(ns frontend.schema.handler.common-config
+  "Schema that is common for global-config and repo-config"
+  (:require [malli.util :as mu]))
+
+(def Config-edn
```

Comment:
This now serves a nice, compact overview of what config options are available. Hopefully we keep this up to date. I thought about adding a test that would keep this in sync with templates/config.edn but decided against it for now